README.md

```diff
@@ -1 +1,18 @@
 # Node.js Inclusivity Slack Inviter
+
+Automated [Slack](https://slack.com/) invitations for the [Node.js Inclusivity WG](https://github.com/nodejs/inclusivity).
+
+## Configuration
+
+Configuration settings are stored in `config.json`. The following settings must be defined:
```

Comment:
Yes, though if we want to support env vars, we can just use dotenv so that we don't need to document two different configuration methods.